GUI/Types/Renderer/MaterialLoader.cs

```diff
@@ -88,7 +88,14 @@ bool TryBindTexture(RenderMaterial mat, string name, string path)
             {
                 if (mat.Shader.GetUniformLocation(name) != -1)
                 {
-                    mat.Textures[name] = LoadTexture(path);
+                    if (name != "g_tSkyTexture")
```

Comment:
Is checking name going to work for cubemap entities too?

Reply:
I don't plan to hardcode this. I think I will end up adding (targetType, handle) as values to the Textures dictionary.